Lib/test/test_sysconfig.py

```diff
@@ -386,11 +386,11 @@ def test_triplet_in_ext_suffix(self):
             self.assertTrue('linux' in suffix, suffix)
         if re.match('(i[3-6]86|x86_64)$', machine):
             if ctypes.sizeof(ctypes.c_char_p()) == 4:
-                self.assertTrue(suffix.endswith('i386-linux-gnu.so') or
-                                suffix.endswith('x86_64-linux-gnux32.so'),
-                                suffix)
+                expected_suffixes = 'i386-linux-gnu.so', 'x86_64-linux-gnux32.so', 'i386-linux-musl.so'
             else: # 8 byte pointer size
-                self.assertTrue(suffix.endswith('x86_64-linux-gnu.so'), suffix)
+                expected_suffixes = 'x86_64-linux-gnu.so', 'x86_64-linux-musl.so'
+        self.assertTrue(suffix.endswith(expected_suffixes),
+                        f'unexpected suffix {suffix!r}')
```

Comment:
I'm totally a neophyte here, but I think the buildbot failures might be due to this?  This appears like it should've been indented to be inside the `if re.match(...` block, ala:

```suggestion
            self.assertTrue(suffix.endswith(expected_suffixes),
                            f'unexpected suffix {suffix!r}')
```

Reply:
ah, that would be it, only one level of dedent was expected!